pkg/provider/azure_loadbalancer.go

```diff
@@ -1498,7 +1498,7 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 			klog.V(2).Infof("reconcileLoadBalancer: reconcileLoadBalancer for service(%s): lb(%s) - updating", serviceName, lbName)
 			err := az.CreateOrUpdateLB(service, *lb)
 			if err != nil {
-				klog.Errorf("reconcileLoadBalancer for service(%s) abort backoff: lb(%s) - updating", serviceName, lbName)
+				klog.Errorf("reconcileLoadBalancer for service(%s) abort backoff: lb(%s) - updating: %s", serviceName, lbName, err.Error())
```

Comment:
how about using error unwrapper instead?

Reply:
do you mean %w?

Reply:
yes

Reply:
but it's in a log, not a constructed error

Reply:
have you come across other usage like this?

Reply:
both formats work. it is ok as long as the error message is clear enough